ex_webrtc/integration_test/test_videoroom/test/integration/simulcast_test.exs

```diff
@@ -20,10 +22,12 @@ defmodule TestVideoroom.Integration.SimulcastTest do
   # time needed to request and receive a variant
   @variant_request_time 2_000
 
+  # TODO: right now noop_variant_selector is used so probing is not used
+  @probe_times %{low_to_medium: 30_000, low_to_high: 45_000, nil_to_high: 50_000}
+
```

Comment:
I guess we could remove this and add once we use probing. For now we could expect immediate switching of variants (limited by the time it takes to request keyframe)